examples/traffic_prediction.jl

```diff
@@ -0,0 +1,96 @@
+# Temporal graph neural networks for traffic prediction
```

Comment:
please expand a bit this comment with the description of the example and add bibliographic references